webapp/src/features/rsuUpdateMenu/RsuUpdateMenu.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { confirmAlert } from "react-confirm-alert";
+import RsuUpdateItem from "../../components/RsuUpdateItem";
+import "react-confirm-alert/src/react-confirm-alert.css";
+import {
+  selectChecked,
+  selectOsUpdateAvailable,
+  selectFwUpdateAvailable,
+
+  // actions
+  updateRsuData,
+  performOSUpdate,
+  performFWUpdate,
+} from "./rsuUpdateMenuSlice";
+import { useSelector, useDispatch } from "react-redux";
+
+import "./rsuUpdateMenu.css";
+
+const RsuUpdateMenu = (props) => {
+  const dispatch = useDispatch();
+  const checked = useSelector(selectChecked);
```

Comment:
checked doesn't seem to be used, do we need it?

Reply:
removed